webpack.config.prod.js

```diff
@@ -1,22 +1,12 @@
-const webpack = require("webpack");
-const UglifyJs = require("uglifyjs-webpack-plugin");
 const path = require("path");
 const baseConfig = require("./webpack.config.base");
 
 module.exports = Object.assign({}, baseConfig, {
+  mode: "production",
   bail: true,
   output: Object.assign({}, baseConfig.output, {
     filename: "[name]-[chunkhash].js",
     path: path.resolve("./dist"),
   }),
-  plugins: baseConfig.plugins.concat([
-    new UglifyJs({}),
-    new webpack.DefinePlugin({
-      "process.env.NODE_ENV": JSON.stringify("production"),
-    }),
-    new webpack.optimize.CommonsChunkPlugin({
-      name: "vendor",
-      filename: "vendor-[chunkhash].bundle.js",
-    }),
-  ]),
+  plugins: baseConfig.plugins.concat([]),
```

Comment:
All the old production plugins we were using are now default behaviors in Webpack 4